web/app/webpack.config.js

```diff
@@ -1,6 +1,7 @@
 const path = require('path');
 
 module.exports = {
+  devtool: 'cheap-module-source-map',
```

Comment:
maybe uncomment line 11 and set that to `cheap-module-source-map`, so we don't have this key in twice. 
I originally uncommented it when I wanted sourcemaps, but leaving the cheap one in is also fine with me!